getting-started/templates/Openshift-GCP/os_gcp_supplemental_values.yaml

```diff
@@ -0,0 +1,286 @@
+global:
+  ingress:
+    api:
+      annotations:
+        kubernetes.io/ingress.class: <user-defined-ingress-class> #<ATTENTION> Enter the volume name where the token is mounted
+    ui:
+      annotations:
+        kubernetes.io/ingress.class: <user-defined-ingress-class> #<ATTENTION> Enter the volume name where the token is mounted
+
+rabbitmq:
+  global:
+    compatibility:
+      openshift:
+        adaptSecurityContext: disabled
+
+webserver:
+  redis-cluster:
+    global:
+      compatibility:
+        openshift:
+          adaptSecurityContext: disabled
+
+testmonitorservice:
```

Comment:
You're also going to need to set the service's postgres connection string to target the sidecar

Reply:
The 'connectionInfo' has been added which targets the sidecar